tests/testsuite/artifact_dep.rs

```diff
@@ -1636,8 +1651,9 @@ fn dep_of_artifact_dep_same_target_specified() {
     p.cargo("check -Z bindeps")
         .masquerade_as_nightly_cargo(&["bindeps"])
         .with_stderr_data(str![[r#"
-[LOCKING] 2 packages to latest compatible versions
+[LOCKING] 3 packages to latest compatible versions
 [COMPILING] baz v0.1.0 ([ROOT]/foo/baz)
+[COMPILING] one v0.1.0 ([ROOT]/foo/one)
```

Comment:
The problem with this is it's non-deterministic whether `bar` or `one` get compiled first, which means this output causes the test to fail half the time. Looking for a way around this.

Reply:
Maybe you can try `with_stderr_contains`.

Reply:
We're trying to de-emphasize the `conains` cases. 

You can call `.unordered()` on the `str![]`